src/Microsoft.Extensions.Logging.Abstractions/NullLogger.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging.Abstractions.Internal;
+
+namespace Microsoft.Extensions.Logging.Abstractions
+{
+    /// <summary>
+    /// Minimalistic logger that does nothing.
+    /// </summary>
+    public class NullLogger : ILogger
+    {
+        public static NullLogger Instance { get; } = new NullLogger();
+
+        private NullLogger()
+        {
+        }
+
+        /// <summary>
```

Comment:
You could use `<inheritdoc />` for implementations. Should suffice in this case.